README.md

```diff
@@ -1,75 +1,26 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
+# Portafolio: Mobil Version
+This is a repo dedicated to share with you my experience
 
-Additional description about the project and its features.
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+- HTML, CSS
+- Git
 
 ## Authors
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
+👤 **Carlos Herver Solano**
 
-Feel free to check the [issues page](../../issues/).
+- GitHub: [@carloshs1994](https://github.com/carloshs1994)
```

Comment:
## [OPTIONAL]
- [ ] I think it would be nice to add more contacts in the `Author` section of your `README`, I would suggest adding the `twitter` and `LinkedIn` profile so that visitors can get your contacts easily. just like :point_down: 
```
- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)

```
I know you can do it :clap: :clap: